### ISSUE TITLE
14.2 SugarCRM OVA file not importing into VMware (manifest invalid checksum)

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#827]
As reported in [the forums](https://www.turnkeylinux.org/forum/support/mon-20170508-2026/sugarcrm-business-social-crm-software) the v14.2 SugarCRM fails to import on VMware as the checksum is invalid in the manifest.

**Comment by JedMeister** [comment]
FWIW I can't quite reproduce this.

The OVA imports without error in VIrtualBox (double click OVA, or manual import within VBox).

The OVA also manually imports fine into VMware's Workstation Player (12.5.5 build-5234757). However, if I just double click the OVA (with VMware assigned as the default app) I get an error:
```
Unable to open /home/user/v14.2/turnkey-drupal8-14.2-jessie-amd64.ova as a Virtual Machine:

VMX file is corrupt.
```
However, TBH that doesn't make sense as the OVA doesn't even have a VMX?! 

I have no idea what is going on here...

**Comment by JedMeister** [comment]
I've discovered the issue. We use the new version of OVFTool (v4.2). That now uses SHA256 for the file checksums (which is a good thing). However, older versions of VMware products only support SHA1.

I will test including both an SHA1 sum as well as an SHA256 sum and see how that goes. Hopefully we can build in backwards compatibility whilst maintaining that advantage of SHA256 checksums.

In the meantime, the workaround options for older VMware product users are:

 - update to a newer VMware product version (or a different hypervisor or product altogether) 
 - use the VMX from the legacy VMDK build
 - install from ISO
 - or use an older version of the OVFTool (v4.1 or earlier) to convert the OVA to an older format which your VMware will accept.

**Comment by JedMeister** [comment]
Closing this as it was fixed early on in v14.2 dev.
